h2o-core/src/main/java/water/ExternalFrameWriterClient.java

```diff
@@ -150,28 +179,7 @@ public void sendDenseVector(double[] values) throws IOException {
         sendBoolean(ExternalFrameUtils.VECTOR_IS_DENSE);
         ExternalFrameUtils.sendDoubleArray(ab, channel, values);
     }
-
-    /**
-     * This method ensures the application waits for all bytes to be written before continuing in the control flow.
-     *
-     * It has to be called at the end of writing.
-     * @param timeout timeout in seconds
-     * @throws ExternalFrameConfirmationException
-     */
-    public void waitUntilAllWritten(int timeout) throws ExternalFrameConfirmationException {
```

Comment:
not required for the same reason as mentioned https://github.com/h2oai/h2o-3/pull/3817/files#r315192609